### PR TITLE
Contribution info and tooltip showing donation/giftcard breakdown 

### PR DESCRIPTION
**Pull request by Jeremy-Xue**
New PR since the last one had a decent amount of stuff refactored (moved from ownerpanel component to progressbar)

Sellers page
Screen Shot 2020-05-08 at 8 32 35 PM

Merchant Directory
Screen Shot 2020-05-08 at 8 32 45 PM

Merchant Directory (hovering over the # contributors -- not sure why wing hing's image isn't showing up on my local)
Screen Shot 2020-05-08 at 8 32 52 PM

### CODE REVIEW

File: src/components/OwnerPanel/OwnerPanel.tsx [hunk 1]
```diff
@@ -174,6 +184,37 @@ const OwnerPanel = (props: Props) => {
         </div>
       }
 
+      {/* hide extra info section until needed */}
```
Comment (Jeremy-Xue) [inline src/components/OwnerPanel/OwnerPanel.tsx:187]: Just noticed this, is this okay to remove? I think it was here from a while ago

Comment (jtmckibb) [inline src/components/OwnerPanel/OwnerPanel.tsx:187]: What is this?

File: src/components/ProgressBar/ProgressBar.tsx [hunk 2]
```diff
@@ -32,9 +54,45 @@ const ProgressBar = ({
...
+        </div>
+        <div>
+          <SupporterTooltip
+            title={
+              <React.Fragment>
+                <table>
+                  <tbody>
+                    <tr>
+                      <td>
+                        <b>{numGiftCards}</b> gift cards
```
Comment (jtmckibb) [inline src/components/ProgressBar/ProgressBar.tsx:70]: This should say "vouchers" — but keep the variable to say numGiftCards

**Comment by nanxiy** [comment]
> Looks good! Is the owner panel also supposed to have the hover state too? @nanxiy

yeah there should be a hover state, it's on the Figma but might not have been clear, I've added it to the payments page

**Pull request merged**